src/app/pages/buy/buy.layout.tsx

```diff
@@ -27,19 +38,18 @@ export const BuyLayout = (props: BuyLayoutProps) => {
         px={['loose', 'unset']}
         spacing="base"
       >
-        <Box width={['100px', '115px']}>
-          <img src={AddFunds} />
-        </Box>
-        <PageTitle mt={['unset', 'base']}>Fund your account</PageTitle>
+        {desktopViewport ? (
+          <Box width={['100px', '115px']}>
+            <img src={AddFunds} />
+          </Box>
+        ) : null}
+
```

Comment:
If we only show this on desktop viewports, there no need for media queries on the width, the first will do nothing

Reply:
`<Box width=115px>` works instead?